tests/test_relational_operand.py

```diff
@@ -13,12 +13,12 @@ def setup():
     """
     module-level test setup
     """
-    A().insert(A.contents, skip_duplicates=True)
-    L().insert(L.contents, skip_duplicates=True)
-    B().populate()
-    D().populate()
-    E().populate()
-    Experiment().populate()
+    A.insert(A.contents, skip_duplicates=True)
```

Comment:
We should mix in instance based inserts as that's supposed to work too. In general, please do not remove test cases but rather just add.

Reply:
Yes, we have a good mixture.  If you review the tests, the majority of tests still instantiate the classes.

Reply:
As in every operation that's supposed to work for either instance or class ought to be tested on both. And it still holds that in general you shouldn't just be replacing previous test cases if they are supposed to continue to work.

Reply:
This is not meant for testing, just the regular setup.  It works.

Reply:
We do have testing for both instances and classes.